src/KIM/pair_kim.cpp

```diff
@@ -762,7 +757,24 @@ void PairKIM::set_argument_pointers()
   int kimerror;
   kimerror = KIM_ComputeArguments_SetArgumentPointerDouble(
     pargs, KIM_COMPUTE_ARGUMENT_NAME_coordinates, &(atom->x[0][0]));
-
+  // Set KIM pointer appropriately for Energy
```

Comment:
@yafshar I think you need to handle the case where the KIM Model has the `partialEnergy` argument as `required` (i.e., a non-null pointer *must* be set).  Currently, it seems to assume the model has `partialEnergy` as `optional`.

Reply:
@ellio167 Fixed it.